packages/bpk-component-calendar/src/Week.js

```diff
@@ -106,7 +106,9 @@ function getSelectionType(
 
   if (
     selectionConfiguration.type === CALENDAR_SELECTION_TYPE.single &&
-    selectionConfiguration.date === formatDateFull(date)
+    selectionConfiguration.date &&
+    (selectionConfiguration.date === formatDateFull(date) ||
+      formatDateFull(selectionConfiguration.date) === formatDateFull(date))
```

Comment:
added this as currently in our documentation we use as an example for `formatDateFull` a function that takes a date and outputs a string with this format `EEEE, do MMMM yyyy`. `selectionConfiguration.date` should normally be a date object so then they would never be equal. I didn't want to change this directly in case there are consumers that are doing changes so that they are equal because in that case it could break for them, but I think this way if it's broken, it will get fixed. If it's not, it will continue to work